lib/src/__tests__/test-utils.tsx

```diff
@@ -66,8 +66,8 @@ const getUtilClass = () => {
   }
 };
 
-export const UtilClassToUse: any = getUtilClass();
-export const utilsToUse: IUtils<MaterialUiPickersDate> = new UtilClassToUse();
+export const DateAdapter: any = getUtilClass();
```

Comment:
Cool, one step in the right direction for #1969.

Reply:
I see the MaterialUiPickersDate type was removed in the next branch.  Thoughts on typing the useState in the Form wrapper?

Reply:
See #1969